spec/models/principal_spec.rb

```diff
@@ -203,6 +203,25 @@
     end
   end
 
+  describe '#onboarded?' do
+    context 'when a principal has been onboarded' do
+      before :each do
+        principal.firm.update_column(:email_address, 'acme@example.com')
```

Comment:
It's a shame this email address thing is still spreading around the codebase. I wonder what we can do to encapsulate it better?